frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -141,6 +142,17 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     combinedDisk.getStorageClassName() || '',
   );
 
+  React.useEffect(() => {
+    if (!isEditing) {
+      const defaultStorageClass = getLoadedData(storageClasses, []).find(
+        (sc) => getAnnotations(sc, {})[DEFAULT_SC_ANNOTATION] === 'true',
+      );
+
+      setStorageClassName(getName(defaultStorageClass) || '');
```

Comment:
let's check if not empty before calling it

Reply:
checking :t-rex: